test/quantization/core/test_quantized_tensor.py

```diff
@@ -1032,7 +1038,7 @@ def test_qtensor_masked_fill(self):
                 [numel], scale=scale, zero_point=zero_point, device=device,
                 dtype=qtype)
             # mask fill the whole tensor, equivalent to calling plain vanilla fill
-            mask = torch.tensor(True)
+            mask = torch.tensor(True).to(device)
```

Comment:
could also be ` torch.tensor(True, device = device)`?

Reply:
Full version ensuring dtype would be `torch.tensor(True, device=device, dtype = torch.float32)`, right? @albanD, isn't 
 torch.Tensor as constructor (implicitly using default tensor type) is deprecated in favor of more explicit and verbose torch.tensor(...) constructor?